docs/docs/learn/introduction/gateway/index.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 2
+sidebar_label: "BOB Gateway: Bitcoin Intents"
+---
+
+# BOB Gateway: Bitcoin Intents
+
+## Introduction
+
+Bitcoin users can easily onboard to the BOB Hybrid L2 without previously holding any Ethereum assets. This page explains the structure of _BOB Gateway_, an intent-based bridge that coordinates peer-to-peer swaps between users and liquidity providers (LPs).
+
+Cross-chain transfers are secured by verifying Bitcoin transaction proofs with an [on-chain light client](/learn/builder-guides/relay), avoiding the need for an oracle. Optional intents, such as staking, lending, and swapping a small amount of ETH for transaction fees can all be accomplished while only requiring a single Bitcoin transaction from the user.
+
+## Next Steps for Gateway
+
+We are interested in working closely with builders looking to connect their smart contracts as new _strategies_ for BOB Gateway users.
+
+These are some of the features we're working on for Gateway's next upgrade, with new ideas being added frequently.
+
+- "Unwrap" your wrapped BTC on BOB by withdrawing back to BTC on Bitcoin mainnet
+- "Unstake" your BTC LST/LRT back to BTC on Bitcoin mainnet
+- Swapping from BTC to any ERC20 through a DEX (already possible, not fully implemented yet)
+
+:::info Integrate BOB Gateway Into your App
+[See our Builder Guide](/learn/builder-guides/gateway) to learn more about our SDK and extending BOB Gateway's functionality to bring Bitcoiners one transaction away from your use-case.
+:::
+
+## How Gateway Works
+
+1. Liquidity providers (LPs) temporarily lock wrapped Bitcoin (WBTC or tBTC) in escrow smart contracts on BOB.
+1. A user makes a request to the off-chain relayer to reserve some of the available liquidity.
+1. The user sends BTC to the liquidity provider's Bitcoin address. A hash of the user's order is included in the `OP_RETURN` of their transaction, including data such as the recipient's EVM address on BOB.
+1. The relayer trustlessly verifies the user's Bitcoin transaction by submitting a Merkle proof to an on-chain [Light Client](/learn/builder-guides/relay.md), granting the relayer permission to withdraw the LP's wrapped Bitcoin without needing to use an oracle.
+1. Gateway sends the LP's wrapped Bitcoin to the user's EVM address. If the user requested a Bitcoin LST/LRT, that token is minted using the LP's wrapped Bitcoin _before_ it is sent to the user.
+
+## Architecture
+
+![architecture](./architecture.png)
```

Comment:
Any chance we can make that picture a bit larger? Hard to read. Also, you can export a dark theme friendly figure by uploading the source file to excalidraw and exporting a dark themed figure.